electron/src/lib/CoreProtocol.test.main.ts

```diff
@@ -0,0 +1,33 @@
+import * as assert from 'assert';
+import sinon from 'sinon';
+import {CustomProtocolHandler} from './CoreProtocol';
+import {EVENT_TYPE} from "./eventType";
```

Comment:
```suggestion
import {EVENT_TYPE} from './eventType';
```

Reply:
Reminds me to update the linting script...

Reply:
Maybe merge from staging first 😉

Reply:
I see what you did there (https://github.com/wireapp/wire-desktop/pull/2426) but we have "mocha" here, not "jest". ;)

Reply:
Fixed in https://github.com/wireapp/wire-desktop/pull/2433. I was more talking about the automatic linting on commit.